src/gdpr/ConsentManagementPlatform.js

```diff
@@ -96,6 +96,8 @@ class ConsentManagementPlatform {
 
         if (this.isOnWhiteListedPage()) {
             return false;
+        } else if (window.navigator.globalPrivacyControl) {
```

Comment:
```suggestion
        } else if (this.geoRequiresTrackingConsent() && window.navigator.globalPrivacyControl) {
```
And `this.geoRequiresTrackingConsent()` can be stored inside a new variable like `hasConsentCookie`